src/week-view.h

```diff
@@ -31,6 +31,7 @@ void week_view_remove_calendar(WeekView* wv, Calendar* cal);
 int week_view_get_week(WeekView* wv);
 icaltime_span week_view_get_current_view(WeekView* wv);
 void week_view_previous(WeekView* wv);
+void week_view_current(WeekView* wv);
 void week_view_next(WeekView* wv);
```

Comment:
On the topic of better naming, perhaps these functions could be `week_view_go_*` or `week_view_goto_*`